src/main/java/com/byznass/tiolktrack/dao/plsql/PlsqlGpsProvider.java

```diff
@@ -0,0 +1,30 @@
+package com.byznass.tiolktrack.dao.plsql;
+
+import com.byznass.tiolktrack.kernel.dao.GpsProvider;
+import com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException;
+import com.byznass.tiolktrack.kernel.model.Gps;
+import com.byznass.tiolktrack.kernel.model.Location;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+public class PlsqlGpsProvider implements GpsProvider {
+
+	@Override
+	public Gps getGpsById(String gpsId) throws NoGpsWithIdException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)